### PR TITLE
Try to locate the mypy.ini find in the project root.

### PR DESCRIPTION
**Pull request by JulienPalard**
See https://github.com/msherry/flycheck-pycheckers/issues/3

I'm currently trying it, it can be improved as the search for the project root is done twice, it's currently "just a test".

### CODE REVIEW

**Comment by msherry** [comment]
This is a great start -- thanks for taking it on! I always provide the path to the mypy.ini file in the `.pycheckers` config file, so I haven't bothered to implement this yet, but this is definitely a great feature to add.

I think ideally we would look for this file by default, and allow any user-specified options to override it. What do you think?

**Comment by JulienPalard** [comment]
Why not, or searching for files mypy is typically using (mypy.ini with fallback to setup.cfg in the current directory, or .mypy.ini)? I think it could be added later as needed.

File: bin/pycheckers.py [hunk 1]
```diff
@@ -559,6 +560,10 @@ def get_run_flags(self, filepath):
                     filepath)
             flags += ['--config-file', self.options.mypy_config_file]
```
Comment (msherry) [inline bin/pycheckers.py]: My point was that we’re using the user-supplied option here on line 560 by default, but then immediately throwing that away if we find a file in the vcs root. We should never discard explicit user options, so we should only check for files in the default location if the user has not provided this option (and avoid passing `—config-file` twice).

Something like
```
if user_option_supplied:
   ...
else if mypy_ini_in_vcs_root:
   ...

**Comment by JulienPalard** [comment]
@msherry is that better?

**Comment by msherry** [comment]
Oops, this fell off my radar -- I apologize for that. Yes, this looks good now! Thanks for making the requested fixes.

**Pull request merged**